reference-architecture/aws-ansible/playbooks/roles/s3-registry-bucket/tasks/main.yaml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: set the name of the bucket
-  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry"
+  set_fact: s3_bucket="{{ ansible_date_time.epoch }}-openshift-docker-registry-{{ stack_name}}"
```

Comment:
@detiber I added the epoch time here due to the S3 buckets must be unique regardless of region. So for example if an org would use prod-openshift-infra and then another org blindly uses the same thing then the installation will fail at the s3 point.  

Also, the s3 bucket creation is so far down in the deployment that I would think the role wouldn't be re-applied.